src/server/helpers/achievement.js

```diff
@@ -222,16 +279,12 @@ function processSprinter(editorId) {
 		});
 }
 
-function achievementToUnlockId(achievementUnlock) {
-	const unlockIds = [];
-	Object.keys(achievementUnlock).forEach((key) => {
-		if (achievementUnlock[key].id) {
-			unlockIds.push(String(achievementUnlock[key].id));
-		}
-	});
-	return unlockIds;
-}
-
+/**
+ * Gets number of distinct days the editor made an edit on within set limit
```

Comment:
"made an edit on within set limit" could be replaced with "created revisions within specified time period"